core/spec/models/spree/user_spec.rb

```diff
@@ -42,7 +42,7 @@
     context "persists order address" do
       let!(:order) { create(:order, bill_address: create(:address), ship_address: create(:address)) }
 
-      it "copies over order addresses" do
+      def addresses_were_cloned
```

Comment:
I could but I dislike them and we prefer this approach instead. They clobber your stacktrace, making it really hard to see where a failure occurs. And I don't see how they are better. But I'd be happy to learn otherwise.